homework/insensitive-palindrom/palindrome.cpp

```diff
@@ -0,0 +1,11 @@
+#include "palindrome.hpp"
+#include <algorithm>
+#include <cctype>
+
+bool is_palindrome(std::string text) {
+    text.erase(std::remove_if(text.begin(), text.end(), [](unsigned char c)
+                              { return isspace(c) || !isalpha(c); }),
+               text.end());
+    std::transform(text.begin(), text.end(), text.begin(), tolower);
+    return std::equal(text.cbegin(), text.cend(), text.crbegin(), text.crend());

```

Comment:
You need to check only half of string

...................|
Ala ma kota atok am ala